Emscripten 6 added OpenMP support, so autoconf's AC_OPENMP check now succeeds
and healpix_cxx's configure appends -fopenmp to CFLAGS/CXXFLAGS. On Emscripten
`-fopenmp` implies pthreads which is causes a link error.

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -173,6 +173,7 @@
                 "--disable-maintainer-mode",
                 "--without-zlib-check",
                 "--without-fortran",
+                "--disable-openmp",
             ]
 
             log.info("%s", " ".join(cmd))
